public/js/pgcal.js

```diff
@@ -71,20 +82,24 @@ document.addEventListener("DOMContentLoaded", function () {
       }
     },
 
-    initialView: pgcal_is_mobile() ? listType : "dayGridMonth",
-
     // Change view on window resize
     windowResize: function (view) {
       // Catch mobile chrome, which changes window size as nav bar appears
       // so only fire if width has changed.
-      if (window.innerWidth !== width) {
+      if (
+        window.innerWidth !== width &&
+        views.hasList &&
+        views.wantsToEnforceListviewOnMobile
+      ) {
         if (pgcal_is_mobile()) {
-          calendar.changeView(listType);
-        } else {
-          calendar.changeView("dayGridMonth");
+          selectedView = calendar.view.type
+          return calendar.changeView(views.listType);
```

Comment:
This was misbehaving as a return, so I set it back to a straight `calendar.changeView(views.listType); Could have been something else I was tweaking, but is working now. 

What was the reason for adding the `return`?

Reply:
No reason at all. It passed my tests here (but it shouldn't - just saw it's type explicitly returns void). I usually don't like else statements on my methods, so I return the main value/behavior at the end and return alternative values before, on each condition. I find it more readable.